src/Compilers/CSharp/Portable/Parser/BlendedNode.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
 {
-    internal struct BlendedNode
+    internal readonly struct BlendedNode
```

Comment:
❓ What prevents this in C# 7.2? If it's allowed, it would be helpful in the review if the C# 7.2 features could be brought in separately from the C# 7.3 features.

Reply:
I did not realize that 7.2 is no longer barred. I will separate 7.3 into a separate PR, so that this could actually get merged.

---
In reply to: [179906078](https://github.com/dotnet/roslyn/pull/26010#discussion_r179906078) [](ancestors = 179906078)

Reply:
Note that I still need to update the toolset compiler packages to 2.7.0  which matches `Visual Studio 2017 version 15.6` 
There was a number of important fixes and it seems the changed code is relying on some of them.

---
In reply to: [180535550](https://github.com/dotnet/roslyn/pull/26010#discussion_r180535550) [](ancestors = 180535550,179906078)